### DISCUSSION TITLE
Compile to WebAssembly?

### DISCUSSION THREAD

**Discussion by LukePrior** [clangd/clangd#1099] [Q&A]
Hi all,

I was wondering if anybody has had any success compiling clangd to webassembly so that it can be run in the browser client-side and provide C/C++ language server support to a Monaco editor. I know that this has been discussed in the past but I'm not sure if anyone has managed to create such a prototype.

**Comment by HighCommander4** [comment]
Looks like someone is working on building LLVM for WebAssembly, which is a first step towards this: https://discourse.llvm.org/t/rfc-building-llvm-for-webassembly/79073

**Comment by HighCommander4** [comment]
See also https://github.com/guyutongxue/clangd-in-browser
